### PR TITLE
Remove Pydb integration as Pydb is unmaintained and not ion PyPI

### PR DESCRIPTION
**Pull request by Carreau**
Closes # 9482

### CODE REVIEW

File: IPython/core/interactiveshell.py [hunk 1]
```diff
@@ -974,7 +974,7 @@ def _set_call_pdb(self,val):
                         'Control auto-activation of pdb at exceptions')
 
     def debugger(self,force=False):
-        """Call the pydb/pdb debugger.
+        """Call the /pdb debugger.
```
Comment (takluyver) [inline IPython/core/interactiveshell.py]: Leftover /

File: docs/source/whatsnew/pr/remove-pydb.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+Integration with pydb has been removed since pydb development has been stopped since 2012, and pydb is not installable from PyPI
```
Comment (takluyver) [inline docs/source/whatsnew/pr/remove-pydb.rst]: I think this should be an `incompat-` file, so it winds up in the incompatible changes section of the release notes.

**Comment by takluyver** [comment]
I think this is fine - no need to maintain extraneous complexity.

**Comment by Carreau** [comment]
Done.

**Pull request merged**